core/src/main/scala/cats/instances/package.scala

```diff
@@ -9,12 +9,12 @@ package object instances {
   object byte extends ByteInstances
   object char extends CharInstances
   object double extends DoubleInstances
-  object duration extends DurationInstances
+  object duration extends CoreDurationInstances with DurationInstances
```

Comment:
I'm still trying to solve an issue here as importing both `cats.instances.duration._` and `cats.instances.finiteDuration._` explicitly prevents for the instance to be resolved. The scenario is when there's the need for kernel instances together with core's
```scala
object test {

  import cats.instances.duration._
  import cats.instances.finiteDuration._

  implicitly[Order[Duration]]
  implicitly[Order[FiniteDuration]]

  implicitly[ContravariantShow[Duration]]
  implicitly[ContravariantShow[FiniteDuration]]
}
```

Reply:
I found a way to mitigate the problem. Now the only case that would fail is the below (see [these test cases](https://github.com/typelevel/cats/pull/2670/files#diff-ddba966829745b5540048fba87ca4014R54) as an example).
```scala
test("fails") {
  import cats.instances.duration._
  import cats.instances.finiteDuration._

  implicitly[ContravariantShow[Duration]] // fails compilation
  implicitly[ContravariantShow[FiniteDuration]] // is inferred
}
```
I couldn't find a way to remove also this drawback.